domain/model/state/types.go

```diff
@@ -146,11 +146,17 @@ type dbUUID struct {
 	UUID string `db:"uuid"`
 }
 
-// dbModelUUID represents the model uuid from the model table.
-type dbModelUUID struct {
+// dbModelUUIDRef represents the model uuid from the model table. As referenced
+// by other tables in the DDL. Use this type when you are working model uuid's
```

Comment:
`As referenced by ...` is not a complete sentence.
`uuid's` doesn't need an apostrophe, plus we've been capitalising it, `UUIDs`

I wonder though, this seems excessively verbose for a very obvious struct.

Reply:
Fixed up.